client/my-sites/woocommerce/woop/woop-landing-page.jsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+import Image01 from 'calypso/assets/images/woocommerce/image01.jpeg';
```

Comment:
should we rename the files with something more descriptive? or maybe create a folder?

Reply:
please feel free to ignore this suggestion :-)

Reply:
✅ We already have a folder for this one under `assets/images/woocommerce` but went ahead and changed the name to `woop-cta-imageXX` that is more descriptive.